dominio/src/main/dominio/Mapa.java

```diff
@@ -66,4 +65,21 @@ public void moverBomber(Bomber personaje, double despX, double despY) {
 		}
 
 	}
+	
+	public void explotarBomba(Bomba bomb) {
+		for(Bomber bomber: bombers) {
+			if((Math.abs(bomber.getPosicionX()-bomb.getPosicionX()) <= bomb.getRango()) && (Math.abs(bomber.getPosicionY()-bomb.getPosicionY()) <= bomb.getRango())) {
+				bomber.morir();
+			}
+		}
+		//SE CONSIDERA SOLO EL RANGO 1
```

Comment:
Una solución fácil para considerar todos los rangos es:
Traer el rango una sola vez y despues hacer un for desde 1 hasta el rango inclusive, con las mismas 4 lineas de codigo de acá abajo (reemplazando `bomb.getRango()` por i.

Reply:
Ahora me pongo con eso